deploy/docs/v1_migration_doc.md

```diff
@@ -165,8 +165,15 @@ above the output plugin section [here](https://github.com/SumoLogic/sumologic-ku
 - The unified Fluentd `statefulsets` have been split into set of two different Fluentd's, one for `logs` and the other one for `metrics`.
 - We now support the collection of renamed metrics (for Kubernetes version `1.17+`).
 ### How to upgrade for Non-helm Users
-#### 1. Tear down existing Fluentd, Prometheus, Fluent Bit resources
-Run the commands mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#tear-down) to delete the respective resources.
+#### 1. Tear down existing Fluentd, Prometheus, Fluent Bit and Falco resources
+You will need the YAML files you created when you first installed collection. Run the following commands to remove Falco, Fluent-bit, Prometheus Operator and FluentD.  You do not need to delete the Namespace and Secret you originally created as they will still be used.
+
+```sh
+kubectl delete -f falco.yaml
+kubectl delete -f fluent-bit.yaml
+kubectl delete -f prometheus.yaml
+kubectl delete -f fluentd-sumologic.yaml
```

Comment:
Should we care about the Prometheus CRDs?

Reply:
Ahh good point.  If we dont, we get the CRD's already exist error.

Reply:
Is that still true? On my 1.18 cluster I get the following:
```
$ kubectl delete -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io "podmonitors.monitoring.coreos.com" deleted

$ kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com created

$ kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com configured

$ kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com configured
```

Reply:
When I was testing `kube-prometheus-stack` version bump PR https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1089 I remember also being able to just apply new yamls on top of the old ones without an error.

Reply:
I've just checked that with EKS 1.14.9 (the earliest we support) and kubectl version `v1.15.11` and `v1.19.3` and I see `configured` all the time.

Reply:
That's fine until we don't use `kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml`

Reply:
👍  thanks!
As we use `kubectl apply` in our documentation I'm going to not add this info to documentation and just merge the PR.